src/data/prefixDataAccess.ts

```diff
@@ -0,0 +1,70 @@
+import { Knex } from "knex";
+import knex from "./database";
+import { DISCORD } from "../../config.json";
+import logger from "../logging";
+import _ from "lodash";
+
+const guildTableName = "guild";
+
+/**
+ * Sets new Prefix for a guild
+ * @param guildId - The id of the guild
+ * @param newPrefix - The new prefix
+ * @param prefixMaxLength - The maximum length of the prefix
+ * @returns - A promise, that writes to the database
+ */
+const setPrefix = async (guildId: string, newPrefix: string, prefixMaxLength: number = 3): Promise<any> => {
```

Comment:
prefixMaxLength now defined in two files. Here and _guild/prefix.ts_. If it shall be changed, it must be changed in both files -> error-prone